sdk/keyvault/azkeys/utils_test.go

```diff
@@ -31,7 +31,20 @@ var enableHSM = true
 
 func TestMain(m *testing.M) {
 	// Initialize
-	if recording.GetRecordMode() == "record" {
+	switch recording.GetRecordMode() {
+	case recording.PlaybackMode:
```

Comment:
Comment here on why we don't do playback mode?

Reply:
All of these changes are only for the response bodies. There's no affect on request bodies.